troubleshooting-2.md

```diff
@@ -0,0 +1,466 @@
+
+``` r
+filter(movieLens, year == 1999 | year == 2000)
+```
+
+    ## # A tibble: 9,088 × 7
+    ##    movieId title                             year genres userId rating timestamp
+    ##      <int> <chr>                            <int> <fct>   <int>  <dbl>     <int>
+    ##  1    2694 Big Daddy                         1999 Comedy      3    3      1.30e9
+    ##  2    2702 Summer of Sam                     1999 Drama       3    3.5    1.30e9
+    ##  3    2762 Sixth Sense, The                  1999 Drama…      3    3.5    1.30e9
+    ##  4    2841 Stir of Echoes                    1999 Horro…      3    4      1.30e9
+    ##  5    2858 American Beauty                   1999 Drama…      3    4      1.30e9
+    ##  6    2959 Fight Club                        1999 Actio…      3    5      1.30e9
+    ##  7    3510 Frequency                         2000 Drama…      3    4      1.30e9
+    ##  8    3949 Requiem for a Dream               2000 Drama       3    5      1.30e9
+    ##  9   27369 Daria: Is It Fall Yet?            2000 Anima…      3    3.5    1.30e9
+    ## 10    2628 Star Wars: Episode I - The Phan…  1999 Actio…      4    5      9.50e8
+    ## # ℹ 9,078 more rows
+
+``` r
+filter(movieLens, rating > 4.5, year < 1995)
+```
+
+    ## # A tibble: 8,386 × 7
+    ##    movieId title                             year genres userId rating timestamp
+    ##      <int> <chr>                            <int> <fct>   <int>  <dbl>     <int>
+    ##  1     265 Like Water for Chocolate (Como …  1992 Drama…      2      5    8.35e8
+    ##  2     266 Legends of the Fall               1994 Drama…      2      5    8.35e8
+    ##  3     551 Nightmare Before Christmas, The   1993 Anima…      2      5    8.35e8
+    ##  4     589 Terminator 2: Judgment Day        1991 Actio…      2      5    8.35e8
+    ##  5     590 Dances with Wolves                1990 Adven…      2      5    8.35e8
+    ##  6     592 Batman                            1989 Actio…      2      5    8.35e8
+    ##  7     318 Shawshank Redemption, The         1994 Crime…      3      5    1.30e9
+    ##  8     356 Forrest Gump                      1994 Comed…      3      5    1.30e9
+    ##  9    1197 Princess Bride, The               1987 Actio…      3      5    1.30e9
+    ## 10     260 Star Wars: Episode IV - A New H…  1977 Actio…      4      5    9.50e8
+    ## # ℹ 8,376 more rows
+
+While filtering for *all movies that do not belong to the genre drama*
+above, I noticed something interesting. I want to filter for the same
+thing again, this time selecting variables **title and genres first,**
+and then *everything else*. But I want to do this in a robust way, so
+that (for example) if I end up changing `movieLens` to contain more or
+less columns some time in the future, the code will still work. Hint:
+there is a function to select “everything else”…
+
+``` r
+### ERROR HERE ###
+movieLens %>%
+  filter(!genres == "Drama") %>%
+  select(title, genres,everything())
+```
```

Comment:
Good work here! I think this is the intended answer with `everything()` based on _"so that (for example) if I end up changing `movieLens` to contain more or less columns some time in the future, the code will still work."_